client_test.go

```diff
@@ -7,6 +7,9 @@ import (
 	"context"
 	"reflect"
 	"testing"
+	"strings"
+	rollbarErrors "github.com/rollbar/rollbar-go/errors"
```

Comment:
This would add a dependency on rollbar-go/errors to the module rollbar-go and therefore defeat the purpose of a separate module.

Reply:
Tests for the errors sub module must be in that or another module and not the main module, otherwise the main module would inherit all the sub modules dependencies again.

Reply:
OK thank you. I thought it would not, if it were only in the test file.

Reply:
See the following wiki page for more information about this: https://github.com/golang/go/wiki/Modules#why-does-go-mod-tidy-record-indirect-and-test-dependencies-in-my-gomod

Reply:
Thank you for the link.

Reply:
Removed. Not sure what's the best approach, but no longer feels like a good fit for this PR.

Reply:
I think it would make sense to have a 3rd module called e.g. "integration" were some integration tests could live. This module could depend on rollbar-go and rollbar-go/errors.